scoping.md

```diff
@@ -1,20 +1,27 @@
 # Scoping
 
 ## Members
+
 Whole Team
 
-Prep: [Create Basecamp Project](https://www.notion.so/companycam/Cycle-Setup-221ad631b5a34e3e8e8302e53b03e1f8)
+# Prep
+
+[Create Basecamp Project](https://www.notion.so/companycam/Cycle-Setup-221ad631b5a34e3e8e8302e53b03e1f8)
```

Comment:
At this point I normally create the retro board so I can link it on the Basecamp project.

Reply:
I also create the project results tracker item.